doc/source/about.rst

```diff
@@ -0,0 +1,110 @@
+
+Within the Met Office IMPROVER is being used to provide a new probabilistic approach to post-processing forecasts. Gridded forecasts taken from multiple configurations of the Met Office's Unified Model (UM) first undergo physical corrections if necessary, before being converted into probabilities. The resulting probabilistic forecasts from each UM configuration are then blended to create seamless probabilistic outputs. These gridded probabilities aim to deliver both improved forecast information at each grid square and at geographical (“spot”) locations.
+
+Key principles of the Met Office's post processing strategy that are being built into IMPROVER are:
+
+  1. A separate processing chain for each variable (e.g. temperature, rain rate) which operates on a  grid with location-specific information extracted at the end.
+  2. A fully probabilistic system that allows for time-lagging and blending between models/ensembles and provides seamless probabilistic forecasts that exploits the benefits of each of the models used.
+  3. The ability to provide outputs to enable automated forecasts of “ordinary” weather for any required location as well as more granular information about the possibility of high-impact weather for constructing warnings and risk estimation.
+  4. A modular software framework following modern professional software development practices.
+
+An example IMPROVER forecast product is shown below, here for wind speed. Four fields are plotted at different wind speed thresholds that may be of interest to the user, each field shows the probability of exceeding the relevant threshold.
+
+
+.. figure:: ../files/wind_probabilities.jpg
+   :align: center
+
+   Probability of exceeding wind speed thresholds
+
+
+Structure of IMPROVER
+---------------------
+
+IMPROVER is designed as a modular post-processing system. The final product is created through the application of a sequence of processing steps, where the sequence can be readily modified to achieve different outcomes. The output of each step in the chain can be written out, allowing for verification against observations at each stage (using a suitable verification package, which is not part of IMPROVER). This enables the user to determine whether a given step in the chain is improving or harming the forecast quality.
+
+Any given step in the processing can be applied using the included command line interfaces (CLIs). A complex system is built by calling the CLIs in sequence. The simple schematic below gives an example processing chain.
```

Comment:
Add link from CLIs to "Using IMPROVER" paragraph.

Reply:
Done.